### PR TITLE
 [#ICC-293] Update servicemessage template with placeholder

### PR DESCRIPTION
**Pull request by fabriziopapi**
Add placeholder to service message template to be replaced with data coming from the message content and from the sender.

### CODE REVIEW

File: Service_Message_WithOrgLogo/index.html [hunk 1]
```diff
@@ -326,32 +328,29 @@
...
-                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:regular;line-height:24px;text-align:left;color:#17324D;">
-                                    <!-- Body 1 Desktop (from MUI Italia)-->
-                                    <p> Buone notizie! <strong>La tua Carta Giovani Nazionale è attiva</strong> e la potrai trovare all’interno della sezione Portafoglio. </p>
-                                    <p> Ti ricordiamo che solo tu puoi usufruire degli sconti presentando la Carta presso gli esercenti fisici aderenti, oppure inserendo i codici nell’apposito campo presso gli esercenti aderenti online. </p>
-                                    <p> La Carta Giovani Nazionale sarà valida da oggi fino al compimento dei 36 anni. Inizia subito a usarla! </p>
-                                  </div>
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:regular;line-height:24px;text-align:left;color:#17324D;"><!-- Body 1 Desktop (from MUI Italia)--> ${body}</div>
```
Comment (dmnplb) [inline Service_Message_WithOrgLogo/index.html:370]: We should make sure that the `${body}` placeholder is replaced with the correct paragraph tags. For example, if we have two different paragraphs:
```html
<p>First paragraph</p>
<p>Second paragraph</p>
```
This is essential if we want the mail to render correctly on different email clients.

File: Service_Message_WithOrgLogo/index.mjml [hunk 2]
```diff
@@ -71,20 +70,7 @@
...
-            <p>
-              Ti ricordiamo che solo tu puoi usufruire degli sconti presentando
-              la Carta presso gli esercenti fisici aderenti, oppure inserendo i
-              codici nell’apposito campo presso gli esercenti aderenti online.
-            </p>
-            <p>
-              La Carta Giovani Nazionale sarà valida da oggi fino al compimento
-              dei 36 anni. Inizia subito a usarla!
-            </p>
+            ${body}
```
Comment (dmnplb) [inline Service_Message_WithOrgLogo/index.mjml:81]: Same as above

**Comment by thisisjp** [comment]
@fabriziopapi can we make an internal test, so that we can approve this story? it's been 2 years 🍰 😆

**Comment by fabriziopapi** [comment]
> @fabriziopapi can we make an internal test, so that we can approve this story? it's been 2 years 🍰 😆

sure, we'll release this update behind a feature flag

File: Service_Message_WithOrgLogo/index.html (hunk 1 above)
Comment (fabriziopapi) [reply Service_Message_WithOrgLogo/index.html:370]: Could be acceptable the use of `<br>` tag to manage the line breaks?
```html
<p>First paragraph<br>
Second paragraph</p>
```

Comment (fabriziopapi) [reply Service_Message_WithOrgLogo/index.html:370]: this option is not feasable due to client compatibility

**Pull request merged**